src/styles/Count.css

```diff
@@ -16,6 +16,11 @@
 }
 
 .iconCountWrapper {
+    display: inline-block;
```

Comment:
https://github.com/scitedotai/scite-widget/pull/34/files#diff-8443734de2fb3f4668886b839c1f9b0630e696b5e74b4b993fcb1a30ed0099b2L19

this is the previous diff